config/opal_setup_cc.m4

```diff
@@ -23,6 +23,119 @@ dnl
 dnl $HEADER$
 dnl
 
+AC_DEFUN([OPAL_CC_HELPER],[
+    AC_MSG_CHECKING([$1])
+
+    AC_COMPILE_IFELSE([AC_LANG_PROGRAM([$3],[$4])], [$2=yes], [$2=no])
+
+    AC_MSG_RESULT([$$2])
+])
+
+
+AC_DEFUN([OPAL_PROG_CC_C11_HELPER],[
+    OPAL_VAR_SCOPE_PUSH([opal_prog_cc_c11_helper_CFLAGS_save opal_prog_cc_c11_helper__thread_local_available opal_prog_cc_c11_helper_atomic_var_available opal_prog_cc_c11_helper__atomic_available opal_prog_cc_c11_helper_tmp opal_prog_cc_c11_helper__static_assert_available])
+
+    opal_prog_cc_c11_helper_CFLAGS_save=$CFLAGS
+    CFLAGS="$CFLAGS $1"
+
+    OPAL_CC_HELPER([if $CC $1 supports C11 thread local storage], [opal_prog_cc_c11_helper__thread_local_available],
+                   [],[[static _Thread_local int  foo = 1;++foo;]])
+
+    OPAL_CC_HELPER([if $CC $1 supports C11 atomic variables], [opal_prog_cc_c11_helper_atomic_var_available],
+                   [[#include <stdatomic.h>]], [[static atomic_long foo = 1;++foo;]])
+
+    OPAL_CC_HELPER([if $CC $1 supports C11 _Atomic keyword], [opal_prog_cc_c11_helper__atomic_available],
```

Comment:
Should this be `opal_prog_cc_c11_helper__Atomic_available` (capital A)?  Ditto for Generic, Static.